build.gradle

```diff
@@ -101,6 +101,10 @@ tasks.withType(Test) {
   }
 }
 
+tasks.withType(Copy) {
+  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Any reason you added this?

Reply:
I thought it seemed reasonable and it fixed the build

https://docs.gradle.org/current/javadoc/org/gradle/api/file/DuplicatesStrategy.html <If you'd like me to choose a different one please let me know. But having duplicates of stuff around isn't always the best...